lib/rules/time-min-milliseconds/__tests__/index.js

```diff
@@ -0,0 +1,211 @@
+"use strict"
+
+const messages = require("..").messages
+const ruleName = require("..").ruleName
+const rules = require("../../../rules")
+
+const rule = rules[ruleName]
+
+const MIN_VALUE = 100
+
+testRule(rule, {
+  ruleName,
+  config: [MIN_VALUE],
+
+  accept: [ {
+    code: "a { transition-delay: 0.1s; }",
+  }, {
+    code: "a { transition-delay: 0.2s; }",
+  }, {
+    code: "a { -webkit-transition-delay: 0.2s; }",
+  }, {
+    code: "a { tRaNsItIoN-dElAy: 0.2s; }",
+  }, {
+    code: "a { TRANSITION-DELAY: 0.2s; }",
+  }, {
+    code: "a { transition-delay: 3s; }",
+  }, {
+    code: "a { transition-delay: 100ms; }",
+  }, {
+    code: "a { transition-delay: 200ms; }",
+  }, {
+    code: "a { transition-delay: 5000ms; }",
+  }, {
+    code: "a { transition-duration: 0.15s; }",
+  }, {
+    code: "a { -webkit-transition-duration: 0.15s; }",
+  }, {
+    code: "a { transition-duration: 2.8s; }",
+  }, {
+    code: "a { transition-duration: 330ms; }",
+  }, {
+    code: "a { transition-duration: 4700ms; }",
+  }, {
+    code: "a { transition: foo 0.8s linear; }",
+  }, {
+    code: "a { -webkit-transition: foo 0.8s linear; }",
+  }, {
+    code: "a { tRaNsItIoN: foo 0.8s linear; }",
+  }, {
+    code: "a { TRANSITION: foo 0.8s linear; }",
+  }, {
+    code: "a { transition: foo 0.8s 200ms ease-in-out; }",
+  }, {
+    code: "a { animation-delay: 0.2s; }",
+  }, {
+    code: "a { -webkit-animation-delay: 0.2s; }",
+  }, {
+    code: "a { animation-delay: 3s; }",
+  }, {
+    code: "a { animation-delay: 200ms; }",
+  }, {
+    code: "a { animation-delay: 5000ms; }",
+  }, {
+    code: "a { animation-duration: 0.15s; }",
+  }, {
+    code: "a { -webkitanimation-duration: 0.15s; }",
+  }, {
+    code: "a { animation-duration: 2.8s; }",
+  }, {
+    code: "a { animation-duration: 330ms; }",
+  }, {
+    code: "a { animation-duration: 4700ms; }",
+  }, {
+    code: "a { animation: foo 0.8s linear; }",
+  }, {
+    code: "a { -webkit-animation: foo 0.8s linear; }",
+  }, {
+    code: "a { animation: foo 0.8s 200ms ease-in-out; }",
+  }, {
+    code: "a { animation-delay: -2.5s; }",
```

Comment:
This is a little confusing and needs to be clarified in the docs. Do *any* negative values work? Or is the "minimum" actually an *absolute* value applying on both sides of zero?